components/camel-coap/src/main/java/org/apache/camel/coap/CoAPEndpoint.java

```diff
@@ -405,19 +420,20 @@ public static boolean enableTCP(URI uri) {
 
     public DTLSConnector createDTLSConnector(InetSocketAddress address, boolean client) throws IOException {
 
-        DtlsConnectorConfig.Builder builder = new DtlsConnectorConfig.Builder();
+        DtlsConnectorConfig.Builder builder = new DtlsConnectorConfig.Builder(Configuration.getStandard());
+        CertificateConfigurationHelper certificateConfigurationHelper = new CertificateConfigurationHelper();
```

Comment:
That's not generally needed. Usually the DtlsConnectorConfig.Builder creates one automatically and applies that to the certificate provider and verifier. The idea behind this is, if remote implementations are used, traversing the credentials may fail and so someone can pass in the default values according the requirements. If local "stores" are used, then the default traverse them and collect the default parameters automatically. 
I would remove it here. 
Otherwise, if you want to keep it, you would need to apply it to the certificate provider and verifier in order to get the defaults filled (see ConfigurationHelperSetup).